src/frontends/pytorch/src/op/shape_as_tensor.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
```

Comment:
should we include specific header instead of whole opset?

Reply:
Yes, @PiotrKrzem  please include just a header of individual operation, for simple operations it is better to not include whole opset

Reply:
Done!